examples/spark/src/test/java/edu/snu/nemo/examples/spark/MRJava.java

```diff
@@ -61,16 +61,16 @@ public void testSparkWordCount() throws Exception {
         .build());
 
     try {
-      ExampleTestUtil.ensureOutputValidity(fileBasePath, outputFileName, testResourceFilename);
+      ExampleTestUtil.ensureOutputValidity(fileBasePath, outputFileName, expectedOutputFilename);
     } finally {
       ExampleTestUtil.deleteOutputFile(fileBasePath, outputFileName);
     }
   }
   /* TODO #152: enable execution of multiple jobs (call scheduleJob multiple times with caching).
   @Test(timeout = TIMEOUT)
   public void testSparkWordAndLineCount() throws Exception {
-    final String inputFileName = "sample_input_wordcount_spark";
-    final String outputFileName = "sample_output_word_and_line_count";
+    final String inputFileName = "test_input_wordcount_spark";
+    final String outputFileName = "test_output_word_and_line_count";
     final String testResourceFilename = "test_output_word_and_line_count";
```

Comment:
`outputFilename` and `testResourceFilename` became the same!
Maybe `testResourceFilename` to `expectedOutputFilename` as above?